app/views/posts/partials/show/_image.html.erb

```diff
@@ -1,3 +1,7 @@
 <% if policy(post).visible? %>
-  <%= image_tag(post.file_url_for(CurrentUser.user), width: post.image_width_for(CurrentUser.user), height: post.image_height_for(CurrentUser.user), id: "image", class: "fit-width", "data-original-width": post.image_width, "data-original-height": post.image_height, "data-large-width": post.large_image_width, "data-large-height": post.large_image_height, "data-tags": post.tag_string, alt: post.presenter.humanized_essential_tag_string, "data-uploader": post.uploader.name, "data-rating": post.rating, "data-flags": post.status_flags, "data-parent-id": post.parent_id, "data-has-children": post.has_children?, "data-has-active-children": post.has_active_children?, "data-score": post.score, "data-fav-count": post.fav_count, "itemprop": "contentUrl") %>
+  <%= content_tag(:picture) do -%>
+    <%= tag.source media: "(min-width: 660px)", srcset: post.tagged_file_url -%>
```

Comment:
Wouldn't this cause the original url to be show for users with `user.default_image_size == "large"`?
Couldn't you just leave this one out entirely and let the browser fall back on the `<img>` tag below?

Reply:
> Wouldn't this cause the original url to be show

Yeah, I missed that.

> Couldn't you just leave this one out entirely and let the browser fall back on the <img> tag below?

To be honest, I was only copying what was being done on post previews for consistency. However, if it's fine to do away with that, I can go ahead and remove that.

Reply:
Yeah actually, doing that fixed the issue of double image loading that I noted in [my prior comment](#issuecomment-640105366), even for Chrome.